src/main/resources/public/jsx/Panel.jsx

```diff
@@ -62,13 +62,13 @@ class Panel extends Component {
         const { t } = this.context
         const log = this.props.log
         return (
-            <div className={'panel ' + t('css_class.panel.' + log.synchronizerAuditLog.status)}>
+            <div className={'panel' + (log.synchronizerAuditLog.status == 'SUCCEEDED'  ? ' panel-success' : log.synchronizerAuditLog.status == 'FAILED' ? ' panel-danger' : ' panel-warning' )}>
                 <div className="panel-heading">
                     <div className="row">
                         <div className="col-md-6">
                             <h3 className="panel-title">
                                 {log.dcModelMapping.resourceName}
-                                <span className={'glyphicon ' + t('css_class.glyphicon.' + log.synchronizerAuditLog.status)} aria-hidden="true"></span>
+                                <span className={'glyphicon ' + (log.synchronizerAuditLog.status == 'SUCCEEDED' ? ' glyphicon-ok' : log.synchronizerAuditLog.status == 'FAILED' ? ' glyphicon-warning-sign' : ' glyphicon-time' )} aria-hidden="true"></span>
```

Comment:
It could have stayed simpler
```
const cssStatus = [
    'SUCCEEDED': 'panel-success',
    'FAILED': 'panel-danger',
    'PENDING': 'panel-warning',
    'MODIFIED': 'panel-warning'
]
className={'panel' + cssStatus[log.synchronizerAuditLog.status]}
```